packages/inscription-view/src/components/parts/common/permission/Permission.test.tsx

```diff
@@ -2,41 +2,72 @@ import { CollapsableUtil, SelectUtil, render, screen } from 'test-utils';
 import { Permission } from './Permission';
 import { describe, test, expect } from 'vitest';
 
-const defaultConfig = { anonymous: true, error: 'Everybody', role: 'ivy:security:forbidden' };
+const defaultConfig = { anonymous: true, error: 'ivy:security:forbidden', roles: ['Everybody'] };
 
 describe('Permission', () => {
-  function renderPart(anonymousFieldActive: boolean, anonymous: boolean, error: string, role: string) {
+  function renderPart(data: PermissionTestData) {
```

Comment:
@ivy-lli I've introduced a value object. I think the tests are now a bit more readable. Let me know if this is not good.

Reply:
Tip top 👍 
Just as info, if you like you can destruct objects directly here to access the different parts in it:
`function renderPart({anonymous, error}: PermissionTestData)`
Then you don't have to write data.anonymous and you can directly write anonymous.